android/src/main/java/com/vydia/RNUploader/UploaderReactPackage.kt

```diff
@@ -0,0 +1,18 @@
+package com.vydia.RNUploader;
+
+import android.view.View
+import com.facebook.react.ReactPackage
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.uimanager.ReactShadowNode
+import com.facebook.react.uimanager.ViewManager
+
+class UploaderReactPackage : ReactPackage {
+    override fun createViewManagers(
+        reactContext: ReactApplicationContext
+    ): MutableList<ViewManager<View, ReactShadowNode<*>>> = mutableListOf()
+
+    override fun createNativeModules(
+        reactContext: ReactApplicationContext
+    ): MutableList<NativeModule> = listOf(UploaderModule(reactContext)).toMutableList()
+}
```

Comment:
Replaced the java version with kotlin, following the RN's Android native modules docs
https://reactnative.dev/docs/native-modules-android?android-language=kotlin